cpp/tests/filling/sequence_tests.cu

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
I think this could be a `.cpp` file as I don't see any device code

Reply:
Following the pattern the other fill tests use (repeat_tests.cu, fill_tests.cu)

Reply:
Definitely prefer a .cpp file if you can as it's faster to compile. Patterns shouldn't be followed if they are bad patterns. Is there any device code in the other tests?

Reply:
No, they're all host-side.